src/components/PermissionsView.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react'
+
+import { translate } from 'cozy-ui/transpiled/react/I18n'
+
+import Page from 'components/Page'
+import PageTitle from 'components/PageTitle'
+
+class PermissionsView extends Component {
```

Comment:
Pour tous les nouveaux composants, préférer les composants fonctionnels :)
Et par la même, utiliser le hook `useI18n` à la place du HOC `translate`

Reply:
Je suggère de pair-programmer pour refactorer. Ça permettra d'effectuer les changements plus vite et aussi d'expliquer les conventions d'équipe en allant. 🙏

Reply:
By the way, si l'un d'entre vous deux peut approuver ma modification de guideline : 🙏 
https://github.com/cozy/cozy-guidelines/pull/75